test/python/test_relugrad.py

```diff
@@ -0,0 +1,49 @@
+# ==============================================================================
+#  Copyright 2018 Intel Corporation
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+# ==============================================================================
+"""nGraph TensorFlow bridge ReluGrad operation test
+
+"""
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import pytest
+
+import tensorflow as tf
+from tensorflow.python.framework import constant_op
+from tensorflow.python.ops.gen_nn_ops import relu_grad
+
+from common import NgraphTest
+
+
+class TestCastOperations(NgraphTest):
+  def test_cast_2d(self):
+    gradients = constant_op.constant([1.5, 2.5, 3.5, 4.5, 5.5, 6.5], shape = [2,3])
```

Comment:
This test is a good one. Just think of some improvement, maybe?  Could we try some random number generation instead of constant input? (this is totally optional)

Reply:
Yes, working on this one.